src/Tools/ConfigurationSchemaGenerator/ConfigSchemaEmitter.cs

```diff
@@ -369,36 +491,46 @@ internal static void AppendUnindentedValue(StringBuilder builder, string value)
         {
             builder.Append(value, index, remaining);
         }
+
+        return builder.ToString();
     }
 
-    private void AppendTypeNodes(JsonObject propertyNode, TypeSpec propertyTypeSpec)
+    private static string GetElementText(XElement element)
     {
-        if (propertyTypeSpec is ParsableFromStringSpec parsable)
+        if (element.Name == "para" || element.Name == "p")
         {
-            AppendParsableFromString(propertyNode, parsable);
+            return $"<br/><br/>{element.Value}<br/><br/>";
         }
-        else if (propertyTypeSpec is ObjectSpec)
-        {
-            propertyNode["type"] = "object";
-        }
-        else if (propertyTypeSpec is EnumerableSpec)
-        {
-            // TODO: support enumerables correctly
```

Comment:
It looks like we lost this TODO and handling for `EnumerableSpec`. Can we add it back?

Reply:
There's no need for that. The behavior is the same for its base type, `CollectionSpec`:

![image](https://github.com/dotnet/aspire/assets/104792814/10193317-c567-442d-b22c-97ef71726e9a)

The test coverage is at `GeneratorTests.ShouldConvertTypes`:
```c#
[InlineData("IEnumerable<string>?",
    """
    "type": "array",
    "items": {
      "type": "string"
    }
    """)]
```

Reply:
Sounds good. Thanks for the explanation.